### ISSUE TITLE
E3002 Invalid Property Resources/GlueJob/Properties/Targets/DynamoDBTargets/0/scanAll

### ISSUE THREAD

**Issue by Jerrys-C** [aws-cloudformation/cfn-lint#2555]
### CloudFormation Lint Version

0.72.2

### What operating system are you using?

Windows

### Describe the bug

AWS::Glue::Crawler MongoDBTarget should allowed scanAll parameter
https://docs.aws.amazon.com/zh_cn/glue/latest/dg/aws-glue-api-crawler-crawling.html

### Expected behavior

scanAll parameter should be allowed in AWS::Glue::Crawler MongoDBTarget

### Reproduction template

```yaml
    GlueCrawlerRequest:
        Type: "AWS::Glue::Crawler"
        Properties:
            Name: !Sub glue-${pDivision}-${pEnv}-${pRegionCode}-${pProjectName}-dmp-crawler-request-${pIteration}
            Role: !Sub ${pLambdaIAMRole}
            Targets:
                DynamoDBTargets:
                  -
                    Path: !Sub ${DynamoDBTableRequest}
                    scanAll: false
            DatabaseName: !Sub glue_${pDivision}_${pEnv}_${pRegionCode}_${pProjectName}_dmp_cs_landing_db_${pIteration}
            SchemaChangePolicy:
                UpdateBehavior: "UPDATE_IN_DATABASE"
                DeleteBehavior: "DEPRECATE_IN_DATABASE"
            Tags:
              "jbl:cost_center": !Ref pCostCenter
              "jbl:owner_email": !Ref pOwnerEmail
              "jbl:project_name": !Ref pTagProjectName
              "jbl:support_team_email": !Ref pSupportTeamEmail
              "jbl:created_date": !Ref pCreateDate

**Comment by kddejong** [comment]
`scanAll` doesn't look like a standard CloudFormation property.  Are you sure this is working as planned?

**Comment by Jerrys-C** [comment]
i think `scanAll` should be a property of DynamoDB target 
https://docs.aws.amazon.com/ja_jp/AWSCloudFormation/latest/UserGuide/aws-properties-glue-crawler-dynamodbtarget.html
this is only `path` in the Documentation for now

**Comment by Jerrys-C** [comment]
i think `scanAll` should be a property of DynamoDB target 
https://docs.aws.amazon.com/ja_jp/AWSCloudFormation/latest/UserGuide/aws-properties-glue-crawler-dynamodbtarget.html
this is only `path` in the [Documentation](https://docs.aws.amazon.com/index.html)

**Comment by kddejong** [comment]
@Jerrys-C shouldn't it be `ScanAll`?  I haven't seen a lower case property before so I'm curious.

**Comment by kddejong** [comment]
I'm getting this error `Property validation failure: [Encountered unsupported properties in {/Targets/DynamoDBTargets/0}: [scanAll]]` when deploying this resource.

If you have a stack with this value deployed I may need to work with you offline to look into a few things.
